scripts/quay-v3.0.4/delete-quay.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
```

Comment:
@ecwpz91  - I see we ported Tosin's changes to upgrade to Quay v3.  are all these quay changes being called from the ansible playbook or is this manual bash script? I must be missing where it is invoked from the playbook.
Also, it seems like some of the quay credential information has changed, does this still require manual configuration? either way we should probably update the instructions in the readme

Reply:
@ecwpz91 clarified that this is just preparatory work to get ready for Quay v3. These aren't being called yet